src/gui/static/src/app/components/pages/exchange/exchange.component.ts

```diff
@@ -1,21 +1,28 @@
 import { Component, OnInit, OnDestroy } from '@angular/core';
-import { StoredExchangeOrder } from '../../../app.datatypes';
-import { ExchangeService } from '../../../services/exchange.service';
 import { MatDialog } from '@angular/material/dialog';
-import { ExchangeHistoryComponent } from './exchange-history/exchange-history.component';
 import { SubscriptionLike } from 'rxjs';
+
+import { ExchangeService, StoredExchangeOrder } from '../../../services/exchange.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
import { ExchangeService, StoredExchangeOrder } from "../../../services/exchange.service";
```